wagtail/wagtailadmin/templates/wagtailadmin/admin_base.html

```diff
@@ -25,6 +25,10 @@
     <script src="{% static 'wagtailadmin/js/vendor/jquery.dlmenu.js' %}"></script>
     <script src="{% static 'wagtailadmin/js/vendor/tag-it.js' %}"></script>
     <script src="{% static 'wagtailadmin/js/core.js' %}"></script>
+    <script>
+        window.taggitSpacesAllowed = {% if TAGGIT_SPACES_ALLOWED %}true{% else %}false{% endif %};
```

Comment:
`{{ TAGGIT_SPACES_ALLOWED|yesno:'true,false' }}` can also be used (not sure which one I prefer, however).